CommandLine/testing/CommonTest.hpp

```diff
@@ -8,7 +8,10 @@
 inline void test_common(TestHarness* test_harness, const std::string& name, bool save_image = true) {
   if (!test_harness) FAIL() << "Game could not be run.";
 
-  test_harness->wait();  // Let the game render a frame first.
+  for (int i = 0; game_running && i < 10; ++i) {
+    test_harness->wait();  // Let the game render a few frames first.
+  }

```

Comment:
That might bust on some of my more complex drawing tests where I carefully choreographed the sequence of events. For example, the room transition test already has a number of steps embedded to test everything before it shuts itself down.

Reply:
mkay. I'll keep that in mind. for now I just wanted to confirm if that's why I'm getting black screen on some images

Reply:
I think you may need to modify the tests to screen_save instead of screenshoting. If I'm correct about issue here.

Reply:
It's just a sleep, anyway. The Window manager is slow. I thought I waited long enough, apparently I didn't. We'll never be able to precisely snapshot frame-by-frame; we'll need the game to export each frame itself, or we'll need to simulate keys instead.

Still, I'd up then wait time instead of adding a for loop.